src/page/GraphEditorPage.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './GraphEditorPage.css';
+import Graph from '../graph/Graph';
+import Timeline from './Timeline';
+
+class GraphEditorPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            vertices: [],
+            edges: [],
+        };
+
+        // array of XY coords
+        this.vertices = [[0, 1], [2, 3], [-3, 2]];
+        // array of FROM TO vertices
+        this.edges = [[0, 1], [0, 2]];
```

Comment:
[MAJOR] We shouldn't hardcode the vertices and edges. The `GraphEditorPage` should know which is the active event at the moment => the event of a click on a specific event in the timeline should be propagated to the `GraphEditorPage` thus it would know what is the current event we are in.
Also I agree that:
- the `Timeline` component should know what is the current event it is in. I think that the `GraphEditorPage` should be passing the data to it via the `props` object. Also that the `Timeline` component should be making a fetch to a back-end to obtain a batch of data for it's surrounding events.
- the `GraphEditorPage` should know what is the current event to be able to draw the current state => it would draw the graph of the state it is within.